### PR TITLE
Jormun: use unicode instead of repr for better log and status format

### PR DESCRIPTION
**Pull request by pbougue**
Also a little cleanup of includes

### CODE REVIEW

File: source/jormungandr/jormungandr/realtime_schedule/cleverage.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def _get_next_passage_for_route_point(self, route_point, count=None, from_dt=Non
         return self._get_passages(route_point, r.json())
 
     def status(self):
-        return {'id': self.rt_system_id,
+        return {'id': unicode(self.rt_system_id),
```
Comment (pbougue) [inline source/jormungandr/jormungandr/realtime_schedule/cleverage.py:148]: @kinnou02 changed it in status also... Can be reverted if not smart (like we want to search from it in redis where we don't use unicode...)

Comment (kinnou02) [reply source/jormungandr/jormungandr/realtime_schedule/cleverage.py:148]: ?

**Review by kinnou02 — APPROVED** [review]

**Comment by pbougue** [comment]
TODO:

- [x] add tests (at least on status)
- [x] add unicode_literals everywhere for safety (and correct decode/encode issues)
- [x] test synthese connector as it uses XML and it has decode/encode issue
- [x] wait the very beginning of a sprint for merge to let the time to crash

File: source/jormungandr/jormungandr/realtime_schedule/cleverage.py (hunk 1 above)
Comment (pbougue) [reply source/jormungandr/jormungandr/realtime_schedule/cleverage.py:148]: OK, stupid remark, we don't care :smile:

**Review by pbougue — COMMENTED** [review]
This PR is now OK to be merged (we still have to wait the beginning of a sprint, though).
:mag_right:  Please, read it by commit, it should be way easier! :beach_umbrella:

**Comment by pbougue** [comment]
Jenkins, test this please

**Comment by pbougue** [comment]
Jenkins, test this please

File: source/jormungandr/jormungandr/realtime_schedule/tests/siri_test.py [hunk 2]
```diff
@@ -150,6 +148,6 @@ def next_passage_for_route_point_failure_test():
         assert passages is None
 
 def status_test():
-    siri = Siri(id='tata', service_url='http://bob.com/', requestor_ref='Stibada')
+    siri = Siri(id='tata-é$~#@*!§èû', service_url='http://bob.com/', requestor_ref='Stibada')
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/realtime_schedule/tests/siri_test.py]: you should add a quote `'`

**Review by kinnou02 — COMMENTED** [review]
I'm not very confident in this :cry:

File: source/jormungandr/jormungandr/realtime_schedule/tests/cleverage_test.py [hunk 3]
```diff
@@ -488,7 +488,7 @@ def next_passage_with_theoric_time_response_test(mock_theoric_response):
 
 
 def status_test():
-    cleverage = Cleverage(id='tata', timezone='Europe/Paris', service_url='http://bob.com/',
+    cleverage = Cleverage(id='tata-é$~#@*!§èû', timezone='Europe/Paris', service_url='http://bob.com/',
```
Comment (xlqian) [inline source/jormungandr/jormungandr/realtime_schedule/tests/cleverage_test.py]: lol :+1:

**Review by TeXitoi — APPROVED** [review]

**Comment by pbougue** [comment]
Jenkins, test this please

**Pull request merged**